hermes/src/store/types.rs

```diff
@@ -0,0 +1,303 @@
+use {
```

Comment:
I think you should split up where these types are defined such that they are close to the code that uses them. This class mixes up code from many different layers of the system (the rest API, the storage system, etc). You typically want to keep those layers separated so they have crisp boundaries.

Reply:
yeah i noted this in the PR description too.